ganja.js

```diff
@@ -640,17 +640,17 @@
               if (b1 && !b2 && !b3) {
                 o = o.Scale(-1/o.Dot(cga2d_ni).s);
                 lx=sc*(o.e1); ly=sc*(-o.e2); lr=0; return res2=`<CIRCLE onmousedown="this.parentElement.sel=${oidx}" cx="${lx}" cy="${ly}" r="${options.pointRadius*0.03||0.03}" fill="${color||'green'}"/>`;
-              } else if (!b1 && !b2 && b3) { var isLine=Element.Coeff(4,1,3,-1).LDot(o).Length==0;
+              } else if (!b1 && !b2 && b3) { var isLine=cga2d_ni.LDot(o).Length==0;
               // Lines.
                 if (isLine) {
-                  var loc=((Element.Coeff(4,-.5).Add(Element.Coeff(3,-.5))).LDot(o)).Div(o), att=(Element.Coeff(4,1,3,-1)).LDot(o);
+                  var loc=((Element.Coeff(4,-.5).Add(Element.Coeff(3,-.5))).LDot(o)).Div(o), att=cga2d_ni.LDot(o);
```

Comment:
Which of `(Element.Coeff(4,-.5).Add(Element.Coeff(3,-.5)` (here) and `Element.Coeff(4,.5,3,.5)` (line 653) is `e0`?

Reply:
```
ni = 1e4 - 1e3
no = .5e4 + .5e3
```
I've apparently used -no too (probably to save some extra negations). 

All of this is up for change tho, as we are currently undertaking an effort to standardize the several Witt basis conventions.

I also wanted to provide support for custom ni and no, will need some time to make the needed changes. And I should change all of them to the extended Coeff syntax and save some operations.